lib/streamer.js

```diff
@@ -0,0 +1,24 @@
+const http = require('http-call').HTTP
+const maxRetries = 10
+
+async function call (url, out, retries) {
```

Comment:
If you made this `call (url, out, retries = 0)` you won't need the extra function

Reply:
The method signatures are different. We don't allow setting the `retries` argument in the exported method.